lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -54,78 +54,7 @@ namespace {
     = { HdTokens->displayColor, HdTokens->displayOpacity, HdTokens->normals };
 
-const MColor       kOpaqueBlue(0.0f, 0.0f, 1.0f, 1.0f); //!< Opaque blue
-const MColor       kOpaqueGray(.18f, .18f, .18f, 1.0f); //!< Opaque gray
-const unsigned int kNumColorChannels = 4;               //!< The number of color channels
-
-const MString kPositionsStr("positions");       //!< Cached string for efficiency
-const MString kNormalsStr("normals");           //!< Cached string for efficiency
-const MString kDiffuseColorStr("diffuseColor"); //!< Cached string for efficiency
-const MString kSolidColorStr("solidColor");     //!< Cached string for efficiency
-
-//! A primvar vertex buffer data map indexed by primvar name.
-using PrimvarBufferDataMap = std::unordered_map<TfToken, void*, TfToken::HashFunctor>;
-
-//! \brief  Helper struct used to package all the changes into single commit task
-//!         (such commit task will be executed on main-thread)
-struct CommitState
-{
-    HdVP2DrawItem::RenderItemData& _renderItemData;
-
-    //! If valid, new index buffer data to commit
-    int* _indexBufferData { nullptr };
-    //! If valid, new primvar buffer data to commit
-    PrimvarBufferDataMap _primvarBufferDataMap;
-
-    //! If valid, world matrix to set on the render item
-    MMatrix* _worldMatrix { nullptr };
-
-    //! If valid, bounding box to set on the render item
-    MBoundingBox* _boundingBox { nullptr };
-
-    //! if valid, enable or disable the render item
-    bool* _enabled { nullptr };
-
-    //! Instancing doesn't have dirty bits, every time we do update, we must update instance
-    //! transforms
-    MMatrixArray _instanceTransforms;
-
-    //! Color parameter that _instanceColors should be bound to
-    MString _instanceColorParam;
-
-    //! Color array to support per-instance color and selection highlight.
-    MFloatArray _instanceColors;
-
-    MStringArray _ufeIdentifiers;
-
-    //! If valid, new shader instance to set
-    MHWRender::MShaderInstance* _shader { nullptr };
-
-    //! Is this object transparent
-    bool _isTransparent { false };
-
-    //! If true, associate geometric buffers to the render item and trigger consolidation/instancing
-    //! update
-    bool _geometryDirty { false };
-
-    //! Construct valid commit state
-    CommitState(HdVP2DrawItem::RenderItemData& renderItemData)
-        : _renderItemData(renderItemData)
-    {
-    }
-
-    //! No default constructor, we need draw item and dirty bits.
-    CommitState() = delete;
-
-    //! returns true if there is no state to commit
-    bool Empty()
-    {
-        return _indexBufferData == nullptr && _shader == nullptr && _enabled == nullptr
-            && !_geometryDirty && _boundingBox == nullptr && !_renderItemData._usingInstancedDraw
-            && _instanceTransforms.length() == 0 && _ufeIdentifiers.length() == 0
-            && _worldMatrix == nullptr;
-    }
-};
+const MColor kOpaqueBlue(0.0f, 0.0f, 1.0f, 1.0f); //!< Opaque blue
```

Comment:
There is already MayaUsdRPrim::kOpaqueBlue so we don't need another version of it here.

https://github.com/Autodesk/maya-usd/pull/2076/files#diff-3be343169a56ae82491707b0790e28079ca0357ce91914123ec0cdd222e9c0c2R156

Reply:
I'll take care of that in the following PR